classes/ActionScheduler_Abstract_QueueRunner.php

```diff
@@ -22,6 +32,9 @@
 	 * @param ActionScheduler_QueueCleaner      $cleaner
 	 */
 	public function __construct( ActionScheduler_Store $store = null, ActionScheduler_FatalErrorMonitor $monitor = null, ActionScheduler_QueueCleaner $cleaner = null ) {
+
+		$this->created_time = microtime( true );
+
```

Comment:
Minor: We're just setting class properties here, so the extra new lines before and after this new statement can be omitted.

Reply:
@JPry then I'd have to make sure the `=` all align, which makes the diff harder to read. This property also serves a different purpose to the others, so I think it's justified it stands alone.